### PR TITLE
completed branch name introduction

### LINKED ISSUE #7: 2.  Introduction
### Repository

- [x] developed on a branch called `introduction`

### HTML

- [x] A section element 
- [x] ... with an introduction to the page's main content
- [x] ... and class "introduction"

### CSS

- [x] the "introduction" class

### PR DESCRIPTION
**Pull request by oguzkarademir**
closes #7

### CODE REVIEW

**Review by aleks2407 — CHANGES_REQUESTED** [review]
Looks good to me, Oguz!
maybe you could align h2 to the center?

**Comment by aleks2407** [comment]
@oguzkarademir , could you please add the link to live in README please?
https://aleks2407.github.io/HYF-TeamPower-project/

File: style.css [hunk 1]
```diff
@@ -1,5 +1,26 @@
+/* Section */
 
-
+.container {
+    width: 80%;
```
Comment (mametur) [inline style.css:9]: if you can add here `margin: auto and height: 525` our website will look great:)

**Comment by mametur** [comment]
@oguzkarademir  also there is to closed body tag, can you also add a title to our website

File: style.css (hunk 1 above)
Comment (oguzkarademir) [reply style.css:9]: I added this changes

**Review by oguzkarademir — COMMENTED** [review]
> @oguzkarademir , could you please add the link to live in README please?
> https://aleks2407.github.io/HYF-TeamPower-project/

I added the link to README

**Review by mametur — APPROVED** [review]
awesome

**Pull request merged**